Remove Linux images (#6)
rancher/fluent-bit-package: README.md

# Packaging for Windows `fluent-bit` Images

[![dockerhub](https://img.shields.io/docker/v/rancher/fluent-bit?sort=semver&style=flat-square)](https://hub.docker.com/r/rancher/fluent-bit/tags)

> This repository houses `fluent-bit` Windows image packaging for Rancher's Logging v2, which uses the [banzaicloud/logging-operator](https://github.com/banzaicloud/logging-operator).

Reviewer: ```suggestion
This repository houses `fluent-bit` Windows image packaging for Rancher's Logging V2, which is based on [banzaicloud/logging-operator](https://github.com/banzaicloud/logging-operator).
```
